src/components/Book.js

```diff
@@ -1,61 +1,67 @@
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { removeBook } from '../redux/books/booksSlice';
 
-const Books = () => {
-  const { books, isLoading } = useSelector((state) => state.books);
+const Book = ({ item }) => {
   const dispatch = useDispatch();
-
-  useEffect(() => {
-    dispatch(fetchBooks());
-  }, [dispatch]);
-
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    const title = document.getElementById('titleInput').value;
-    const author = document.getElementById('authorInput').value;
-    const category = 'non-fiction';
-    const id = Math.random().toString(36).substr(2, 9);
-    const book = {
-      item_id: id, title, author, category,
-    };
-    dispatch(addBook(book));
-  };
-
-  if (isLoading) {
-    return <h2>Loading...</h2>;
-  }
-
   return (
-    <div>
-      <ul>
-        {
-          books.map((book) => (
-            <li key={book.item_id}>
-              <span>{book.title}</span>
-              <span>{book.author}</span>
-              <span>{book.category}</span>
-              <button type="button" onClick={() => dispatch(removeBook(book.item_id))}>Remove Book</button>
-            </li>
-          ))
-        }
-      </ul>
-      <h2>Add New Book</h2>
-      <form>
-        <label htmlFor="titleInput">
-          Title:
-          <input type="text" id="titleInput" name="title" />
-        </label>
-        <br />
-        <label htmlFor="authorInput">
-          Author:
-          <input type="text" id="authorInput" name="author" />
-        </label>
-        <br />
-        <button type="submit" onClick={(e) => handleSubmit(e)}>Add Book</button>
-      </form>
-    </div>
+    <li>
+      <div className="panel">
+        <div className="book-Card">
+          <div className="book-info">
+            <p className="School-of">{item.category}</p>
+            <p className="Book-Title">{item.title}</p>
+            <p className="Book-Author">{item.author}</p>
+            <ul className="ul3">
+              <li>
+                <button className="Comments" type="button">Comments</button>
+              </li>
+              <div className="Line-2" />
+              <li>
+                <button type="button" className="Remove" onClick={() => dispatch(removeBook(item.item_id))}>Remove</button>
+              </li>
+              <div className="Line-2" />
+              <li>
+                <button className="Edit" type="button">Edit</button>
+              </li>
+            </ul>
+          </div>
+          <div className="progress">
+            <div
+              className="progress-bar"
+              aria-valuenow="65"
+              aria-valuemin="0"
+              aria-valuemax="100"
+            />
+            <div className="progress-text">
+              <p className="Percent">
+                57
+                <span>%</span>
+              </p>
+              <p className="Completed">Completed</p>
+            </div>
+          </div>
```

Comment:
- [ ] @gaks1 Great work on your design, kindly ensure the progress bar circle color and thickness match the design given, below is a comparison

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/230619816-0ff157f4-726a-4895-90cc-c6bfebb13429.png) |  ![image](https://user-images.githubusercontent.com/61408860/230619866-f3354ee0-9b33-4261-bc22-bb45ec4877e6.png)